### PR TITLE
Fixed german translation issues #470

### PR DESCRIPTION
**Pull request by Pythonix**
I fixed the issue mentioned in #429 .

Just compare the screenshots below :)
image

image

### CODE REVIEW

File: src/qt/bitcoingui.cpp [hunk 1]
```diff
@@ -1151,7 +1151,7 @@ void BitcoinGUI::setNumConnections(int count)
     default: icon = ":/icons/connect_4"; break;
     }
     labelConnectionsIcon->setPixmap(QIcon(icon).pixmap(STATUSBAR_ICONSIZE,STATUSBAR_ICONSIZE));
-    labelConnectionsIcon->setToolTip(tr("%n active connection(s) to Gridcoin network", "", count));
+    labelConnectionsIcon->setToolTip(tr("%1 active connection(s) to Gridcoin network").arg(count));
```
Comment (denravonska) [inline src/qt/bitcoingui.cpp:1154]: These break existing translations so I think we probably want to keep the old form in Staging and change the source in Development. Or find a way to batch change existing translations.

Comment (Pythonix) [reply src/qt/bitcoingui.cpp:1154]: What do you mean by breaks the existing translations? I tested it with german, english and spanish and it worked well. And this format is already used in the code for example here:
`tooltip += tr("Last received block was generated %1.").arg(text);`

Comment (Pythonix) [reply src/qt/bitcoingui.cpp:1154]: Sorry it was a dumb question, get your point now. It is early in the morning. If I'd write a script to change all the translation strings would that be okay then?

Comment (denravonska) [reply src/qt/bitcoingui.cpp:1154]: If you can whip one up that would solve a lot :)

**Comment by Pythonix** [comment]
Now all language files should be updated.

**Pull request merged**